tests/SConscript

```diff
@@ -0,0 +1,10 @@
+# -*- python -*-
+import os
+
+from lsst.sconsUtils import env, scripts
+scripts.BasicSConscript.tests(pyList=[])
+
+for name in ("OSPL_URI", "OPENSPLICE_LOC"):
```

Comment:
Please also include "ADLINK_LICENSE", since we are using the licensed version for deployment.